torch/lib/THCUNN/generic/PReLU.cu

```diff
@@ -23,11 +23,13 @@ void THNN_(PReLU_updateOutput)(
     input = THCTensor_(newContiguous)(state, input);
 
     int n = THCTensor_(nElement)(state, input);
+    if (input->size[ndim > 1] != nOutputPlane)
+      THError("wrong number of input planes");
```

Comment:
Should it be this?

`THError("wrong number of input planes. Expected %d but got %d.", nOutputPlane, input->size[ndim > 1]);`

Expected should be what we defined in PReLU() module, right?